apps/console/src/features/claims/pages/attribute-mappings.tsx

```diff
@@ -381,6 +462,34 @@ export const AttributeMappings: FunctionComponent<RouteChildrenProps<AttributeMa
                 return panes;
             }
 
+            if (type === ClaimManagementConstants.EIDAS) {
+                const panes: StrictTabProps[ "panes" ] = [];
+
+                ClaimManagementConstants.EIDAS_TABS.forEach((tab: { name: string; uri: string }) => {
+                    const dialect: ClaimDialect = dialects?.find(
+                        (dialect: ClaimDialect) => dialect.dialectURI === tab.uri
+                    );
+
+                    dialect &&
```

Comment:
Shall we use `if` statement instead?

Reply:
Is there a specific reason to do that? Because, currently we are using this method in most of the places.

Reply:
&& is acceptable in conditional rendering and assignment, but in this case, it'd better to use the good ol' if statements for better readability IMO

Reply:
Sure I will change